File: solutions/leetcode/2255-count_prefixes_of_a_given_string.py
class Solution(object):
    def countPrefixes(self, words, s):
        """
        :type words: List[str]
        :type s: str
        :rtype: int
        """
        count = 0
        for word in words:
            if s.startswith(word):
                count += 1
        return count
        

File: solutions/leetcode/2255-count_prefixes_of_a_given_string.cpp
class Solution {
public:
    int countPrefixes(vector<string>& words, string s) {
        int count = 0;
        for (const string& word : words) {
            if (s.find(word) == 0) {
                count++;
            }
        }
        return count;
    }
};